tests/replication/test_federation_sender_shard.py

```diff
@@ -261,7 +261,7 @@ def create_room_with_remote_server(
 
         builder = factory.for_room_version(room_version, event_dict)
         join_event = self.get_success(
-            builder.build(prev_event_ids=prev_event_ids, auth_event_ids=None)
+            builder.build(prev_event_ids=list(prev_event_ids), auth_event_ids=None)
```

Comment:
IIRC there is no defined iteration order for a set. I wondered if this might introduce some non-determinism or similar problem... but it should be fine: the order of the prev events list doesn't have any significance.

Reply:
The order shouldn't matter here is my understanding.